e2e_tests/cli/test_import.py

```diff
@@ -78,15 +80,55 @@ def assert_same_annotation_properties(
             assert expected_property in actual_properties  # type : ignore
 
 
+def get_base_slot_of_item(
+    config_values: ConfigValues, dataset_id: int, item_idx: int
+) -> str:
+    """
+    Returns the base slot for the nth item in a specific `E2EDataset`. The base slot is
+    always the first listed slot
+    """
+    items = list_items(
+        config_values.api_key,
+        dataset_id,
+        config_values.team_slug,
+        config_values.server,
+    )
+    return items[item_idx]["slots"][0]["slot_name"]
+
+
+def assert_annotation_slot_alignment(
+    expected_annotation: dt.Annotation,
+    actual_annotation: dt.Annotation,
+    item_type: str,
+    base_slot: Optional[str],
+) -> None:
+    """
+    Ensures that the slot tied to an `actual_annotation` is aligned depending on the
+    value of `item_type`:
+    - `single_slotted`: Perform no checks
+    - `multi_slotted`: Ensures `actual_annotation.slot_names` is equivalent to
+      `expected_annotation.slot_names`
+    - `multi_channel`: Ensures the `actual_annotation` is tied to the base slot
+    """
+    if item_type == "multi_slotted":
+        if expected_annotation.slot_names:
+            assert expected_annotation.slot_names == actual_annotation.slot_names
+        else:
+            assert actual_annotation.slot_names == [base_slot]
```

Comment:
Trying to think when we have this case: if it's a multislot and I don't specify the slot_name of an annotation then by default it gets uploaded to the base slot?

Reply:
Exactly